cmd/remove.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"fmt"
+	"net"
+
+	operator "github.com/alexellis/k3sup/pkg/operator"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh"
+)
+
+func MakeRemove() *cobra.Command {
+	var command = &cobra.Command{
+		Use:          "remove",
+		Short:        "Remove k3s on a server via SSH",
+		Long:         `Remove k3s on a server via SSH.`,
+		Example:      `k3sup remove --ip 192.168.0.100 --user root`,
+		SilenceUsage: true,
+	}
+
+	command.Flags().IP("ip", net.ParseIP("127.0.0.1"), "Public IP of node")
+	command.Flags().String("user", "root", "Username for SSH login")
+	command.Flags().String("ssh-key", "~/.ssh/id_rsa", "The ssh key to use for remote login")
+	command.Flags().Int("ssh-port", 22, "The port on which to connect for ssh")
+	command.Flags().Bool("sudo", true, "Use sudo for uninstallation. e.g. set to false when using the root user and no sudo is available.")
+	command.Flags().Bool("local", false, "Perform a local uninstall without using ssh")
+
+	command.RunE = func(command *cobra.Command, args []string) error {
+		fmt.Printf("Running: k3sup remove\n")
+
+		local, _ := command.Flags().GetBool("local")
+		ip, _ := command.Flags().GetIP("ip")
+
+		removeK3scommand := " sh /usr/local/bin/k3s-uninstall.sh"
```

Comment:
This isn't the correct command for removing k3s from an agent.

It would be `/usr/local/bin/k3s-agent-uninstall.sh`

See the PR from @yankeexe #285